shared/common-adapters/text.native.js

```diff
@@ -232,6 +238,19 @@ export const styles = {
     fontSize: 18,
     lineHeight: 24
   },
+  textBadge: {
+    ...textCommon,
+    ...globalStyles.fontBold,
+    lineHeight: 11,
+    fontSize: 11
+  },
+  textBadgeAndroid: {
+    position: 'relative',
+    bottom: 3,
+    marginBottom: -3,
+    paddingBottom: 0,
+    lineHeight: 13
```

Comment:
android's positioning is a little different...

I wonder if we should try to correct this in general, but I don't know enough yet if it's a constant error or something that changes depending on size. deferring for now